gojira.sh

```diff
@@ -231,6 +237,7 @@ function parse_args {
   EXTRA_ARGS="${_EXTRA_ARGS[@]}"
   GOJIRA_PORTS="${_GOJIRA_PORTS[@]}"
   GOJIRA_VOLUMES="${_GOJIRA_VOLUMES[@]}"
+  GOJIRA_CLI_ENVS="${_GOJIRA_CLI_ENVS[@]}"
```

Comment:
This conversion is not necessary, as bash can iterate over lists. We were doing this for ports and volumes because we need to export them for another process that will render the `.yml`. This egg seems to happen and be called within the same process I believe.

Reply:
Agreed, I was just trying to follow the process that was there for consistency.